irmaclient/client.go

```diff
@@ -211,6 +213,17 @@ func New(
 	client.initRevocation()
 	client.StartJobs()
 
+	gocron.SetPanicHandler(func(jobName string, recoverData interface{}) {
+		var details string
+		b, err := json.Marshal(recoverData)
+		if err == nil {
+			details = string(b)
+		} else {
+			details = "failed to marshal recovered data: " + err.Error()
+		}
+		client.reportError(errors.Errorf("panic during gocron job '%s': %s", jobName, details))
+	})
+
```

Comment:
Should this panic handler not be set before `initRevocation` is called?

Reply:
Ah yes, of course. Done.